src/root/views/DrefApplicationForm/EventDetails/index.tsx

```diff
@@ -250,6 +251,21 @@ function EventDetails(props: Props) {
             placeholder={strings.whatWhereWhenPlaceholder}
           />
         </InputSection>
+        <InputSection
+          title={strings.uploadPhotos}
+        >
+          <GoFileInput
+            name="images"
+            value={value.images}
+            onChange={onValueChange}
+            accept="image/*"
+            multiple
+            showStatus
+            error={error?.fields?.images}
+          >
+            Upload
+          </GoFileInput>
+        </InputSection>
```

Comment:
Don't we need to show the uploaded file during edit?

Reply:
As discussed, let's create an issue for this later and work on it.